src/hospitals/entities/hospital.entity.ts

```diff
@@ -0,0 +1,19 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity({ name: 'hospitals' })
+export class Hospital {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column()
+  name: string;
+
+  @Column()
+  address: string;
+
+  @Column('text', { array: true })
+  phoneNumbers: string[];
+
+  @Column({ type: 'timestamp without time zone', default: () => 'now()' })
```

Comment:
there is createDateColumn decorator in typeorm that can be used here
https://github.com/typeorm/typeorm/blob/master/docs/decorator-reference.md#createdatecolumn